src/main/java/net/galgus/flink/streaming/connectors/http/HTTPSink.java

```diff
@@ -38,6 +38,9 @@ public void invoke(IN value, Context context) throws Exception {
             writer.close();
 
             int status = conn.getResponseCode();
+            log.info("Status code: " + status);
```

Comment:
change the log to log.info("Status={}", status); and add a more meaningful message